app/models/configuration_script_source.rb

```diff
@@ -1,5 +1,5 @@
 class ConfigurationScriptSource < ApplicationRecord
-  has_many    :configuration_script_payloads
+  has_many    :configuration_script_payloads, :dependent => :nullify
```

Comment:
Yes, meant to change this to `:destroy` after chatting with @bzwei , but got distracted by other bugs.

Now done. Thanks!